environments/python/Dockerfile-2.7

```diff
@@ -2,7 +2,7 @@ FROM alpine:3.5
 
 WORKDIR /app
 
-RUN apk add --no-cache python python-dev build-base py-pip libev-dev libffi-dev && \
+RUN apk add --no-cache python python-dev build-base py-pip libev-dev libffi-dev  gcc bash py-pip && \
```

Comment:
Sorry, why this change is required? Ideally, the environment image should not have to change at all

Reply:
@vishal-biyani Changed this because it was giving error for pip3 command not found
```/bin/sh: pip3: not found
The command '/bin/sh -c apk add --no-cache python3 python3-dev build-base libev-dev libffi-dev &&     pip3 install --upgrade pip &&     rm -r /root/.cache' returned a non-zero code: 127
##[error]Process completed with exit code 127.```

Reply:
@agiwalpooja20 So we are facing same issue in all our builds right now - but this should be fixed as a separate PR anyway. Plus I am not able to reproduce locally when I build the same image - so there is something else going on there.